src/plot_api/plot_config.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
     // we can edit titles, move annotations, etc
     editable: false,
 
-    // plot will respect layout.autosize=true and infer its container size
+    // DO initial autosize
```

Comment:
maybe expand on that a little bit?

e.g. _make plotting code go through auto-size routine regardless of layout.autosize_